tslint.json

```diff
@@ -220,26 +220,26 @@
             "arguments",
             "statements"
         ],
-        // TODO: "curly": true,
-        // TODO: "eofline": true,
+        "curly": false, // changed
+        "eofline": false, // changed
         "import-spacing": true,
         "indent": [
-            false, // TODO: true,
+            true, // TODO: true,
             "spaces"
         ],
         "linebreak-style": false, //changed
         "newline-before-return": false, // changed
-        // TODO: "no-consecutive-blank-lines": true,
+        "no-consecutive-blank-lines": false, // changed
```

Comment:
'no-consecutive-blank-lines' and 'no-trailing-whitespace' are easily auto-fixed and make code easier to read IMO. What's the downside?